util/settings/settings.go

```diff
@@ -445,6 +449,10 @@ const (
 	settingUiBannerURLKey = "ui.bannerurl"
 	// settingUiBannerPermanentKey designates the key for whether the banner is permanent and not closeable
 	settingUiBannerPermanentKey = "ui.bannerpermanent"
+	// settingUiBannerBackgroundColorKey designates the key for the color of the banner background
+	settingUiBannerBackgroundColorKey = "ui.bannerbackgroundcolor"
+	// settingUiBannerTextColorKey designates the key for the color of the banner text
+	settingUiBannerTextColorKey = "ui.bannertextcolor"
```

Comment:
@crenshaw-dev, it appears these changes were incorporated.

Is there any plans on merging this PR after the branch conflicts are resolved, if @Jneville0815 is willing to do so. I believe this would be a great contribution to the banner.